src/main/java/core/basesyntax/service/RegistrationServiceImp.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+
+public class RegistrationServiceImp {
+    private final StorageDao storageDao;
+
+    public RegistrationServiceImp(StorageDaoImpl storage) {
+        this.storageDao = storage;
+    }
+
+    public void register(User user) {
```

Comment:
The register method should return the User object that was added to the storage. This is to ensure that the register method returned the correct user.